herddb-core/src/test/java/herddb/checkpoint/CheckpointCommandTest.java

```diff
@@ -0,0 +1,109 @@
+            manager.start();
+            assertTrue(manager.waitForBootOfLocalTablespaces(10000));
+            execute(manager, "CREATE TABLESPACE 'consistence'", Collections.emptyList());
+            manager.waitForTablespace("tblspace1", 10000);
+            execute(manager, "CREATE TABLE consistence.tsql (k1 string primary key,n1 int,s1 string)", Collections.emptyList());
+            for (int i = 0; i < 10; i++) {
+                execute(manager, "INSERT INTO consistence.tsql (k1,n1 ,s1) values (?,?,?)", Arrays.asList(i, 1, "b"));
+            }
+            execute(manager, "checkpoint consistence", Collections.emptyList());
+        }
+    }
+
+    @Test(expected = herddb.model.TableSpaceDoesNotExistException.class)
+    public void checkpointCommandFailureTest() throws Exception {
+        final String wrongTableSpace = "THIS_TABLESPACE_DOES_NOT_EXISTS";
+        try (DBManager manager = new DBManager("localhost", new MemoryMetadataStorageManager(), new MemoryDataStorageManager(), new MemoryCommitLogManager(), null, null)) {
+            manager.start();
+            assertTrue(manager.waitForBootOfLocalTablespaces(10000));
+            execute(manager, "CREATE TABLESPACE 'consistence'", Collections.emptyList());
+            manager.waitForTablespace("tblspace1", 10000);
+            execute(manager, "CREATE TABLE consistence.tsql (k1 string primary key,n1 int,s1 string)", Collections.emptyList());
+            for (int i = 0; i < 10; i++) {
+                execute(manager, "INSERT INTO consistence.tsql (k1,n1 ,s1) values (?,?,?)", Arrays.asList(i, 1, "b"));
+            }
+            execute(manager, "checkpoint " + wrongTableSpace, Collections.emptyList());
+        }
+    }
+
+    @Test
+    public void checkpointCommandNotEnqueueSuccessTest() throws Exception {
+        final long[] returnedValues = new long[2];
+        try (DBManager manager = new DBManager("localhost", new MemoryMetadataStorageManager(), new MemoryDataStorageManager(), new MemoryCommitLogManager(), null, null)) {
+            manager.start();
+            assertTrue(manager.waitForBootOfLocalTablespaces(10000));
+            execute(manager, "CREATE TABLESPACE 'consistence'", Collections.emptyList());
+            manager.waitForTablespace("tblspace1", 10000);
+            execute(manager, "CREATE TABLE consistence.tsql (k1 string primary key,n1 int,s1 string)", Collections.emptyList());
+            for (int i = 0; i < 10; i++) {
+                execute(manager, "INSERT INTO consistence.tsql (k1,n1 ,s1) values (?,?,?)", Arrays.asList(i, 1, "b"));
+            }
+            Thread first = new Thread(() -> {
+                LOGGER.log(Level.INFO, "First thread has started!");
+                returnedValues[0] = execute(manager, "checkpoint consistence -notEnqueue", Collections.emptyList()).transactionId;
```

Comment:
consistence ?

Reply:
It's a sample tablespace name, used only for testing purposes. Should I change it?

Reply:
got it, thanks.
yes, it is better to use some other name, like "tblspace1" as for other tests

Reply:
do you have test cases about using the name in a case insensitive way ?